setup.py

```diff
@@ -1,9 +1,11 @@
-from setuptools import setup, find_packages
+"""polus-plugins setup.py."""
+
+from setuptools import find_packages, setup
```

Comment:
Since we are upgrading this, let's get rid of setup.py and convert over to using poetry and pyproject.toml.